localhost-http-1.1-server/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Localhost HTTP Server'
+description: 'Starts an HTTP 1.1 server using Python. For TLS connections (including mutual authentication), connect to localhost:4443. For plaintext connections, connect to localhost:8080.'
+
+inputs:
+  root-ca-cert-path:
+    description: "Root CA certificate file path."
+    required: True
+  server-cert-path:
+    description: "Server certificate file path."
+    required: True
+  server-priv-key-path:
+    description: "Server private key file path."
+    required: True
+
+runs:
+  using: "composite"
+  steps: 
+      - name: Install dependencies
+        run: pip install -r $GITHUB_ACTION_PATH/requirements.txt
```

Comment:
Actually about this, but should this be pip3?

Reply:
Nope